substorm-nlp-win.yml

```diff
@@ -73,6 +73,11 @@ dependencies:
     - altgraph==0.17
     - beautifulsoup4==4.9.3
     - cachetools==4.1.1
+    - en-core-web-md==2.3.1
+    - en-rpa-simple==0.0.1
+    - en-rpa-simple-calendar==0.0.3
```

Comment:
```suggestion
    - en-rpa-simple-calendar==0.0.4
```

Reply:
Actually the RPA models should not be checked in the YML at all. As there is not a public repository containing our models. They should all be deleted from it.